cinema/views.py

```diff
@@ -167,6 +183,27 @@ def get_serializer_class(self):
 
         return MovieSessionSerializer
 
+    @extend_schema(
+        parameters=[
+            OpenApiParameter(
+                "date",
+                type={"type": "string", "format": "date",
+                      "items": {"type": "str"}},
+                description="Filter by date (ex. ?date=Y-m-d) "
+                            "or like this (ex. ?date=2024-10-08)",
+            ),
+            OpenApiParameter(
+                "movie",
+                type={"type": "array", "items": {"type": "number"}},
```

Comment:
The 'movie' parameter is defined as an array in the OpenAPI schema, but the code expects a single integer value for filtering. Consider updating the OpenAPI parameter type to 'number' to match the expected input format.